osu.Game/Users/UserActivity.cs

```diff
@@ -25,9 +25,14 @@ public class ChoosingBeatmap : UserActivity
             public override string Status => "Choosing a beatmap";
         }
 
-        public class MultiplayerGame : UserActivity
+        public class MultiplayerGame : SoloGame
```

Comment:
not 100% about this inheritance... feels a bit weird.

Reply:
glossed over this but fair point. should probably be `InGame : UserActivity`, `InSoloGame : InGame`, `InMultiplayerGame : InGame`